master/getting-started/kubernetes/tutorials/advanced-policy.md

```diff
@@ -178,33 +178,34 @@ PING google.com (216.58.219.206): 56 data bytes
 ### Prevent outgoing connections from pods
 
 Kubernetes NetworkPolicy does not provide a way to prevent outgoing connections from pods.  However,
-Calico does. In this section we'll create a Policy using `calicoctl` which prevents all outgoing
-connections from Kubernetes pods in the advanced-policy-demo Namespace.
+Calico does. In this section we'll create egress Policies using `calicoctl`
+which allow the outgoing connections in the advanced-policy-demo Namespace we
+want, all other egress traffic will be denied.
 
-To do this, we'll need to create a Policy which selects all pods in the Namespace, and denies
-traffic that doesn't match another Pod in the Namespace.
+To do this, we'll need to create a Policy which selects all pods in the Namespace, and allows
+egress traffic to other Pods in the Namespace.
 
 ```
 calicoctl apply -f - <<EOF
 apiVersion: v1
 kind: policy
 metadata:
-  name: advanced-policy-demo.deny-egress
+  name: advanced-policy-demo.allow-egress
 spec:
   selector: calico/k8s_ns == 'advanced-policy-demo'
   order: 500
   egress:
-  - action: deny
+  - action: allow
     destination:
-      notSelector: calico/k8s_ns == 'advanced-policy-demo'
+      selector: calico/k8s_ns == 'advanced-policy-demo'
 EOF
 ```
 
 Notice that we've specified an order of 500.  This means that this policy will be applied before any
 of the Kubernetes policies.
 
 We also need to create a policy which allows traffic to access kube-dns. Let's create one now in the kube-system Namespace.
-We'll specify an order of 400 so that it takes precendent over other policies.
+We'll specify an order of 500 so that it takes precendent over the Kubernetes policies also.
```

Comment:
"precendent" is not a word :) .... "precendent" --> "precedence"

I would suggest losing the "also" at the end. It is a little confusing.